### PR TITLE
Giphy code review

### PR DESCRIPTION
**Pull request by dominikslawkowski**

### CODE REVIEW

File: .gitignore copy [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (jakubsikora) [inline .gitignore copy:1]: Why `copy` ?

File: src/api/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import axios from "axios";
+
+const apiUrl = "https://api.livechatinc.com/v3.1";
+
+const getHeaders = (at) => ({
```
Comment (jakubsikora) [inline src/api/index.js:5]: I would just use `accessToken` or `token`.  No need for shortcuts here.

File: src/components/Announcements.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+              onCtaClick();
+            },
+          },
+          remind: {
+            children: "Remind me later!",
+            onClick: toggleOpen,
+          },
+        }}
+      >
+        <h2>Monitore your Agenrs GIFs usage</h2>
```
Comment (jakubsikora) [inline src/components/Announcements.js:43]:
```suggestion
        <h2>Monitor your Agenrs GIFs usage</h2>
```

File: src/components/Announcements.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        <p>
+          New you are able to analyze and monitore what kind of GIFs your Agents
```
Comment (jakubsikora) [inline src/components/Announcements.js:45]:
```suggestion
          New you are able to analyze and monitor what kind of GIFs your Agents
```

File: src/components/BlockList.js [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+  > span {
+    display: inline-block;
+    :not(:last-child) {
+      margin-right: 5px;
+    }
+  }
+`;
+
+const BlockList = () => {
+  const [blockList, setWithBlockList] = useState("turned-on");
```
Comment (jakubsikora) [inline src/components/BlockList.js:46]: If you only have 2 states here, how about just using boolean which would simplify things here.

File: public/index.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!--
```
Comment (arturfracala) [inline public/index.html:13]: These comments are unnecessary. I would delete them to keep all files clean

File: src/components/BlockedPhrase.js [hunk 7]
```diff
@@ -0,0 +1,41 @@
+/** @jsxImportSource @emotion/react */
+import { css, jsx } from "@emotion/react";
+import { useState } from "react";
+import { MdBlock, MdClear } from "react-icons/md";
+
+const containerCss = css`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgb(89, 105, 158);
```
Comment (arturfracala) [inline src/components/BlockedPhrase.js:10]: Could you be persistent with setting colors, please? Sometimes you are using hex values, sometimes sth like this

File: src/components/Suggestions.js [hunk 8]
```diff
@@ -0,0 +1,111 @@
+/** @jsxImportSource @emotion/react */
+import { css, jsx } from "@emotion/react";
+import { useState } from "react";
+import { FormGroup, CheckboxField, RadioButton } from "@livechat/design-system";
+
+const Suggestions = () => {
+  const [shouldLikeOffenUsedGifs, setShouldLikeOffenUsedGifs] = useState(false);
```
Comment (arturfracala) [inline src/components/Suggestions.js:7]: ?
```suggestion
  const [shouldLikeOftenUsedGifs, setShouldLikeOftenUsedGifs] = useState(false);
```

**Review by arturfracala — APPROVED** [review]
Nice work 👏  Can't wait for LCS :D

File: src/views/ChatDetails.js [hunk 9]
```diff
@@ -0,0 +1,272 @@
...
+    if (selectedTab === "search") {
+      const currentGifs = gifs.filter(
+        (_, gifIndex) => gifIndex % numberOfColumns === colIndex
+      );
+
+      return currentGifs;
+    }
+    if (selectedTab === "favourites") {
+      const currentGifs =
+        JSON?.parse(localStorage?.getItem("livechat-giphy-favorites"))?.filter(
```
Comment (jakubsikora) [inline src/views/ChatDetails.js:140]: Why would that be conditional, its a native method?

**Review by jakubsikora — COMMENTED** [review]
Few things here from me. Good work.

My only worry is that there is tons of code. Not sure what's your plan of writing it within the 2h LCS ?

**Pull request closed without merging**